Codeinwp/neve
# chore: make sure starter sites are refreshed when license is toggled

### Summary
Enable starter sites refresh when license is toggled on or off.


### Will affect visual aspect of the product

NO

### Test instructions


- Should be tested with [this PR](https://github.com/Codeinwp/themeisle-onboarding/pull/3).
- Toggle license inside the dashboard
- The starter sites should refresh according to the license tier
- **composer.lock** file should be recommited after we merge [this PR](https://github.com/Codeinwp/themeisle-onboarding/pull/3).


Closes Codeinwp/neve-pro-addon#705.
